src/test/java/rolling/DiceRollerTest.java

```diff
@@ -34,7 +33,7 @@ void setUp() {
                 new Roll("ed", 12, 6),
                 new Roll("cd", 3),
                 new Roll("kd", 1)), Arrays.asList(6, -3), 2);
-        output = Roller.output(result, builder -> builder, "Doom!", 0, true, 12, 15);
+        output = Roller.output(new PointChange(result, 12, 15), builder -> builder, "Doom!", 0, true).getEmbeds();
```

Comment:
Really shouldn't have any Discord Code in Tests.

Reply:
Removed